taprpc/taprootassets.proto

```diff
@@ -999,3 +1006,23 @@ message OutPoint {
     */
     uint32 output_index = 2;
 }
+
+message SubscribeReceiveEventsRequest {
+    // Filter receives by a specific address. Leave empty to get all receive
+    // events for all addresses.
+    string filter_addr = 1;
+}
+
+message ReceiveEvent {
+    // Event creation timestamp.
+    int64 timestamp = 1;
+
```

Comment:
Block height + hash?

Reply:
We don't keep the block hash around in the event, so we wouldn't have that at all times. But I added the block height, maybe that's enough for now?

Reply:
Ah makes sense, yeah I think that's a good stop gap.